### PR TITLE
remove akse linux and pass ossku to cluster create

### PR DESCRIPTION
**Pull request by aegal**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

File: hack/swift/Makefile [hunk 1]
```diff
@@ -9,6 +9,7 @@ AZCLI   ?= docker run -v $(AZCFG):/root/.azure -v $(KUBECFG):/root/.kube -v $(SS
 
 # overrideable variables
 USER       ?= $(whoami)
+OSSKU      ?= Ubuntu
```
Comment (rbtr) [inline hack/swift/Makefile:12]: is this case sensitive?

File: hack/swift/Makefile [hunk 2]
```diff
@@ -84,6 +85,7 @@ byocni-up: rg-up net-up ## Brings up a swift cluster $name in $SUB/$REGION
 		--vnet-subnet-id /subscriptions/$(SUB)/resourceGroups/$(GROUP)/providers/Microsoft.Network/virtualNetworks/$(VNET)/subnets/nodenet \
 		--pod-subnet-id /subscriptions/$(SUB)/resourceGroups/$(GROUP)/providers/Microsoft.Network/virtualNetworks/$(VNET)/subnets/podnet \
 		--generate-ssh-keys \
+		--os-sku $(OSSKU)
```
Comment (rbtr) [inline hack/swift/Makefile:88]: what happens if we don't specify this? jw because I thought that Ubuntu was the default.

**Review by rbtr — COMMENTED** [review]
looks good to me pending a couple of questions

File: hack/swift/Makefile (hunk 1 above)
Comment (aegal) [reply hack/swift/Makefile:12]: Not sure, will test it out though. The documentation had this casing so went with that - https://learn.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az-aks-create

File: hack/swift/Makefile (hunk 2 above)
Comment (aegal) [reply hack/swift/Makefile:88]: It does default to Ubuntu, just passing this parameter to setup the next P.R where I will create it with OS sku of windows. Thought it’s best to separate the P.R’s

Comment (rbtr) [reply hack/swift/Makefile:88]: sounds good

**Review by rbtr — APPROVED** [review]

**Review by estebancams — DISMISSED** [review]

**Comment by rbtr** [comment]
@aegal looks like you've got some syntax errors: https://msazure.visualstudio.com/One/_build/results?buildId=62359894&view=results

**Review by rbtr — CHANGES_REQUESTED** [review]
pipeline fails to run

**Review by rbtr — CHANGES_REQUESTED** [review]

File: .pipelines/singletenancy/aks-swift/e2e-step-template.yaml [hunk 3]
```diff
@@ -32,7 +33,7 @@ steps:
         mkdir -p ~/.kube/
         echo "Create AKS cluster"
         make -C ./hack/swift azcfg AZCLI=az
-        make -C ./hack/swift byocni-up SUB=$(SUB_AZURE_NETWORK_AGENT_TEST) AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(make revision)
+        make -C ./hack/swift byocni-up SUB=$(SUB_AZURE_NETWORK_AGENT_TEST) AZCLI=az CLUSTER=${{ parameters.clusterName }}-${{ parameters.osSku }-$(make revision) OSSKU=${{ parameters.osSku }
```
[the remark this answers is not in the document]
Comment (aegal) [reply .pipelines/singletenancy/aks-swift/e2e-step-template.yaml]: thanks for pointing it out!

**Review by rbtr — APPROVED** [review]

**Pull request merged**